hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -932,15 +932,10 @@ public void testFilterDupes() throws Exception {
     ds2.sync();
     mClient = new HoodieTableMetaClient(jsc.hadoopConfiguration(), tableBasePath, true);
     HoodieInstant newLastFinished = mClient.getCommitsTimeline().filterCompletedInstants().lastInstant().get();
-    assertTrue(HoodieTimeline.compareTimestamps(newLastFinished.getTimestamp(), HoodieTimeline.GREATER_THAN, lastFinished.getTimestamp()
+    // there is not new commit generate for empty commits
```

Comment:
we should actually have this generate an empty commit and test. If we don't then we checkpoints won't move. 
Consider this scenario, when deltastreamer reads from kafka using a custom transformer. If the transformer filters out all records from Kafka, we will have empty input for write, but the kafka offsets have to move ahead.

Reply:
Yes, can understand this. Will add config to control this and by default we allow the empty commits.